Patch to correctly set the MySQL basedir parameter.  This bug has not
yet been submitted upstream.

--- puppetlabs-mysql-3.6.2/manifests/params.pp.orig	2016-04-20 13:31:03.825840809 -0700
+++ puppetlabs-mysql-3.6.2/manifests/params.pp	2016-04-20 13:31:46.763615238 -0700
@@ -1,3 +1,7 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 # Private class: See README.md.
 class mysql::params {
 
@@ -288,7 +292,7 @@
     'Solaris': {
       $client_package_name = 'database/mysql-55/client'
       $server_package_name = 'database/mysql-55'
-      $basedir             = undef
+      $basedir             = '/usr/mysql/5.5/'
       $config_file         = '/etc/mysql/5.5/my.cnf'
       $datadir             = '/var/mysql/5.5/data'
       $log_error           = "/var/mysql/5.5/data/${::hostname}.err"
